### PR TITLE
[EDU-2840] add fields gql ah

### PR DESCRIPTION
**Pull request by hannah-sk**
### Changes

* Added new Activity History dataset fields, descriptions, and examples.

[EDU-2840]: https://aziontech.atlassian.net/browse/EDU-2840?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/en/pages/devtools/api-graphql/features/events-fields/events-fields.mdx [hunk 1]
```diff
@@ -21,9 +21,17 @@ See each available field and their descriptions below.
 | authorEmail | Email address of the Real-Time Manager user who performed the action. Example: `myemail@gmail.com` |
 | authorName | Name of the Real-Time Manager user who performed the action. Example: `Hannah` |
 | comment | Editable space available for users to add comments when performing changes. Example: `Action performed during investigation` |
+| refererHeader | Header of the page from which the API was called. Returns when the API call occurs from an UI. Example: `Test 123` |
```
Comment (bru-andrade) [inline src/content/docs/en/pages/devtools/api-graphql/features/events-fields/events-fields.mdx:24]:
```suggestion
| refererHeader | Header `Referer` from the page from which the API was called. Returns when the API call is made from an UI. Example: `Test 123` |
```
Conferir se está correto

Comment (bru-andrade) [reply src/content/docs/en/pages/devtools/api-graphql/features/events-fields/events-fields.mdx:24]: An UI? Do RTM?

File: src/content/docs/en/pages/devtools/api-graphql/features/events-fields/events-fields.mdx [hunk 2]
```diff
@@ -21,9 +21,17 @@ See each available field and their descriptions below.
...
+| remotePort | Port of the origin that generated the request. Example: `80` |
+| resourceId | Unique identifier of the resource that was created or modified. Example: `8190` |
+| resourceType | Identifier of the resource that was created or modified. Example: `edge_application` |
+| requestData | Data received on the payload of the request generated by the user. Example: `{"test": 123}` |
 | title | Title of the activity, composed of: model name, name, and type of activity. Example: `Pathorigin Default Origin was changed` |
 | ts | Timestamp of when the event was created. Example: `2022-10-20T10:10:10` |
 | type | Type of performed action on Real-Time Manager: `CREATED`, `CHANGED`, `DELETED`, or `SIGNED UP` |
+| userAgent | Header `User-Agent` send in the request. Example: `curl 1.2.6` |
```
Comment (bru-andrade) [inline src/content/docs/en/pages/devtools/api-graphql/features/events-fields/events-fields.mdx:32]:
```suggestion
| userAgent | Header `User-Agent` sent in the request. Example: `curl 1.2.6` |
```

**Review by bru-andrade — APPROVED** [review]

File: src/content/docs/en/pages/devtools/api-graphql/features/events-fields/events-fields.mdx [hunk 3]
```diff
@@ -21,9 +21,17 @@ See each available field and their descriptions below.
 | authorEmail | Email address of the Real-Time Manager user who performed the action. Example: `myemail@gmail.com` |
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/api-graphql/features/events-fields/events-fields.mdx:21]:
```suggestion
| authorEmail | Email address of the user who performed the action in Real-Time Manager. Example: `myemail@gmail.com` |
```

**Review by MarianaAguilera — APPROVED** [review]

**Pull request merged**